tests/e2e/Services/Health/HealthCustomServerTest.php

```diff
@@ -211,6 +211,153 @@ public function testCertificatesSuccess(): array
         return [];
     }
 
+    public function testFunctionsSuccess(): array
+    {
+        /**
+         * Test for SUCCESS
+         */
+        $response = $this->client->call(Client::METHOD_GET, '/health/queue/functions', array_merge([
+            'content-type' => 'application/json',
+            'x-appwrite-project' => $this->getProject()['$id'],
+        ], $this->getHeaders()), []);
+
+        $this->assertEquals(200, $response['headers']['status-code']);
+        $this->assertIsInt($response['body']['size']);
+        $this->assertLessThan(100, $response['body']['size']);
+
+        /**
+         * Test for FAILURE
+         */
+
```

Comment:
Can we remove these comments from all the tests in this file?? Doesn't seem to be essential here.

Reply:
Removed all unnessessary comments about failures in this file